debian/README.debian.md

```diff
@@ -0,0 +1,27 @@
+# Build image
+
+To build the image, run:
+```
+make debian
+```
+
+# Run the container
+
+Run the machine with:
```

Comment:
the title is (almost) correct, the sentence is not :)

One builds an image, then **launches** a container from that image. After that, the container is running on its own until terminated